tutorials/shading/shading_reference/shading_language.rst

```diff
@@ -195,13 +195,13 @@ Arrays
 ------
 
 Arrays are containers for multiple variables of a similar type.
-Note: As of Godot 3.2, only local and varying arrays have been implemented.
+Note: As of Godot 4.0, local, global and varying arrays have been implemented.
```

Comment:
This is unnecessary now. But we should mention in the uniform section that we still don't support uniform arrays.

Reply:
Ok, added